lib/quantcast/index.js

```diff
@@ -177,17 +178,25 @@ Quantcast.prototype.completedOrder = function(track){
 Quantcast.prototype._labels = function(type){
```

Comment:
If I'm understanding correctly, seems like this is doing two things: Taking in and formatting multiple labels, and then formatting/joining those labels. Could we break these into two simpler functions?